src/components/layerlist/LayerListItem.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <!-- Show layer details -->
+  <v-list-group
+    v-if="showDetails"
+    v-model="open"
+    class="text--primary"         
+    >
+    <template v-slot:activator>
+      <v-list-item-action>
+        <v-checkbox
+          color="secondary"
+          hide-details
+          :input-value="layer.getVisible()"
+          @click.capture.stop="onItemClick()"   
```

Comment:
what does this `@click.capture.stop` do?

Reply:
I use this to prevent the event from bubbling up to the v-list-item, where it would toggle the list expansion too.